scripts/fetch-archive.js

```diff
@@ -39,6 +39,25 @@ async function getProjectGitHubData(repo) {
   return { stars: stargazers_count, forks: forks_count, issues: open_issues_count }
 }
 
+async function getPackageSize(npm) {
+  return new Promise((resolve, reject) => {
+    https.get(PACKAGE_SIZE_URL + npm, (res) => {
+      let str = '';
+    
+      res.on('data', (chunk) => {
+        str += chunk;
+      });
+    
+      res.on('end', () => {
+        resolve(JSON.parse(str));
+      });
+    
+    }).on('error', (err) => {
+      reject(error);
+    });
+  });
+}
```

Comment:
We could shrink this a lot by using fetch.

Reply:
Do you already have this as a dependency or do I need to add `node-fetch`?